app/controllers/conferences_controller.rb

```diff
@@ -6,6 +6,9 @@ class ConferencesController < ApplicationController
 
   def index
     @current = Conference.where('end_date >= ?', Date.current).reorder(start_date: :asc)
+    if @current.count == 1
+      redirect_to "/conferences/#{@current.first.id}"
```

Comment:
Did you test the changes in your local machine first? This won't work as routing for conferences is by `short_title` and not `id` .
Also, I suggest using named routes instead. :slightly_smiling_face: